### ISSUE TITLE
Comment says "if we're currently in a 1:1..., start a new chat"

### ISSUE THREAD

**Issue by KentShikama** [matrix-org/matrix-react-sdk#328]
Should I simply change this to "if we're NOT currently in a 1:1..., start a new chat"

or send in the following

```
diff --git a/src/components/views/rooms/MemberInfo.js b/src/components/views/rooms/MemberInfo.js
index ddd0d1f..cc0351d 100644
--- a/src/components/views/rooms/MemberInfo.js
+++ b/src/components/views/rooms/MemberInfo.js
@@ -365,15 +365,9 @@ module.exports = React.createClass({
             var currentRoom = MatrixClientPeg.get().getRoom(this.props.member.roomId);
             currentMembers = currentRoom.getJoinedMembers();
         }
-        // if we're currently in a 1:1 with this user, start a new chat
-        if (currentMembers && currentMembers.length === 2 &&
-            userIds.indexOf(currentMembers[0].userId) !== -1 &&
-            userIds.indexOf(currentMembers[1].userId) !== -1)
-        {
+        if (no_existing_private_one_on_one()) {
             existingRoomId = null;
-        }
-        // otherwise reuse the first private 1:1 we find
-        else {
+        } else { // reuse existing private one on one
             existingRoomId = null;
 
             for (var i = 0; i < rooms.length; i++) {
@@ -391,6 +385,11 @@ module.exports = React.createClass({
                 }
             }
         }
+        function no_existing_private_one_on_one() {
+            return currentMembers && currentMembers.length === 2 &&
+                userIds.indexOf(currentMembers[0].userId) !== -1 &&
+                userIds.indexOf(currentMembers[1].userId) !== -1;
+        }
 

```

**Comment by ara4n** [comment]
oops, it was the comment that was wrong; fixed in bfe50c209c2a44087b4ffdd17a7f32f4c48edc3c.  separately, the code is also broken, but we're working on that: https://github.com/vector-im/vector-web/issues/1613.  thanks for calling it out!

**Comment by ara4n** [comment]
actually, the comment was correct (albeit confusing).  If you are *currently* looking at a 1:1 chat with the user, the 'start chat' button should start a new chat.
